stdlib/LinearAlgebra/src/blas.jl

```diff
@@ -108,6 +108,8 @@ openblas_get_config() = strip(unsafe_string(ccall((@blasfunc(openblas_get_config
     set_num_threads(n)
 
 Set the number of threads the BLAS library should use.
+If n <= 0 then will make no change.
+If n > number of CPU threads, then caps at the number of CPU threads.
```

Comment:
Perhaps "set the number of BLAS threads to be equal to the number of CPU threads"? Caps is a maybe a bit difficult to understand and  should be "cap" if kept to be consistent with "set" and "make".

Reply:
Oh that is much clearer.

```suggestion
If `n`  is greater than number of CPU threads, then the nuumber of BLAS threads wiol be set to the number of CPU threads.
```